cmd/utils/fdlimit_test.go

```diff
@@ -16,12 +16,24 @@
 
 package utils
 
-import "testing"
+import (
+	"fmt"
+	"syscall"
+	"testing"
+)
 
 // TestFileDescriptorLimits simply tests whether the file descriptor allowance
 // per this process can be retrieved.
 func TestFileDescriptorLimits(t *testing.T) {
 	target := 4096
+	var limit syscall.Rlimit
```

Comment:
Please avoid using the `syscall.Rlimit` type here to make the Windows build pass.

Reply:
argh sorry